extension/src/shared/reearth/layers/3dtiles.ts

```diff
@@ -38,6 +44,13 @@ export const TilesetLayer: FC<TilesetProps> = ({
   appearance,
   boxAppearance,
 }) => {
+  const [interactionMode] = useAtom(interactionModeAtom);
+
+  const disabledSelection = useMemo(() => {
+    const mode = interactionMode.value as unknown;
+    return mode === "default" || mode === "move";
+  }, [interactionMode]);
```

Comment:
We should not invoke `atom` in here. You need to pass the interactionMode into [here](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/blob/main/extension/src/shared/layerContainers/general.tsx#L94), and handle the disabledSelection in this hook.